internal/core/src/config/ConfigKnowhere.cpp

```diff
@@ -88,6 +88,16 @@ KnowhereInitSearchThreadPool(const uint32_t num_threads) {
     }
 }
 
+void
+KnowhereInitGPUMemoryPool(const uint32_t init_size, const uint32_t max_size) {
+    if (init_size == 0 && max_size == 0) {
+        knowhere::KnowhereConfig::SetRaftMemPool();
+        return;
+    }
+    knowhere::KnowhereConfig::SetRaftMemPool(size_t{init_size},
```

Comment:
What would happen if one of these variables is equal to zero?

Reply:
If set to 0, the GPU memory pool will directly occupy half of the total GPU memory.